apps/renterd/contexts/config/transformDown.ts

```diff
@@ -186,38 +181,20 @@ export function transformDownPricePinning(
     // Assume the default autopilot named 'autopilot'.
     shouldPinAllowance: p.autopilots['autopilot']?.allowance.pinned || false,
     allowanceMonthPinned: toFixedMaxBigNumber(
-      valuePerPeriodToPerMonth(
-        new BigNumber(p.autopilots['autopilot']?.allowance.value || 0),
-        // If pinned allowance is non zero, the period value will be defined.
-        periodBlocks || weeksToBlocks(6)
-      ),
-      fixedFiat
-    ),
-    shouldPinMaxRPCPrice: p.gougingSettingsPins?.maxRPCPrice.pinned,
-    maxRPCPriceMillionPinned: toFixedMaxBigNumber(
-      valuePerOneToPerMillion(
-        new BigNumber(p.gougingSettingsPins.maxRPCPrice.value)
-      ),
+      new BigNumber(p.autopilots['autopilot']?.allowance.value || 0),
```

Comment:
@ChrisSchinnerl could you confirm:

1. The `autopilots.autopilot.allowance.value` is per month?

2. The UI currently converts autopilot allowance, download estimate, and upload estimates, from per period to per month for display - is it correct that the daemon uses these as per period?

Reply:
The allowance is per period. So if you pin that value to $10 and then later change the duration of a period, you will still only spend $10 per period. So the UI needs to convert that value from $ per period to $ per month itself. 

Upload and download is per TiB uploaded/downloaded
Storage is TB / month (30*144 blocks).
So those fields shouldn't require conversion on your end unless you want to display the bandwidth as TB instead of TiB.

Reply:
Oh yeah that's the expected upload and download per period